tests/python/relay/test_conv2d_nchw_texture.py

```diff
@@ -435,3 +435,641 @@ def test_conv2d_vgg16_winograd_4d():
     graph = build_run_compare(mod, params1, {"data": input_shape}, dtype, target)
     matches = re.findall("winograd", graph)
     assert len(matches) > 0
+
+
+@tvm.testing.requires_opencl
+def test_2conv2d():
```

Comment:
Probably let's rename this test to something more meaningful?

Reply:
removed at all since this test duplicates situation in another test